src/lib/components/navbar/Navbar.component.js

```diff
@@ -48,6 +55,26 @@ const NavbarMenu = styled.div`
   align-items: center;
 `;
 
+const NavbarTabs = styled.div`
+flex: 1;
+  display: flex;
+  justify-content: center;
```

Comment:
It would be nicer to align tabs to left side => `justify-content: flex-start`

Reply:
Sure, will update, right now  :D